File: removeElement.java
import java.lang.reflect.Array;
import java.util.Arrays;
import java.util.Collection;
import java.util.Collections;

class Solution27 {
    public static int removeElement(int[] nums, int val) {

        int sum = 0;
        for (int i = 0; i <nums.length ; i++) {
            if (nums[i]!=val){
                sum++;
            }
            if(nums[i]==val){
                nums[i]='_';
            }
        }

        Arrays.sort(nums);
        return sum;
    }


    public static void main(String[] args) {
        int[] arr = {0,1,2,2,3,0,4,2};
        int target = 2;
        System.out.println(removeElement(arr, target));
    }
}

File: removeElement.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function (nums, val) {

    for (let i=nums.length-1;i>=0;i--) {
        if (nums[i] === val) {
            nums.splice(i,1)
        }
    }

    return nums.length
};


console.log(removeElement([0,1,2,2,3,0,4,2], 2))
